x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/__mocks__/request_responses.ts

```diff
@@ -421,7 +471,7 @@ export const getMockPrivileges = () => ({
     },
   },
   application: {},
-  is_authenticated: false,
+  is_authenticated: true,
```

Comment:
This is the dangerous change from the behaviors below. I would almost recommend putting a `.skip` on it and leaving it as `false` until we fix authentication checks below with some comments about why it is skipped rather than fixing it break to not be correct?

Reply:
https://github.com/elastic/kibana/pull/58292/commits/51909c9b2133c1ee4f4bb39da4271de2acaa1eda